rpm_spec/qubes-mgmt-salt-dom0.spec.in

```diff
@@ -20,6 +20,9 @@ Conflicts:  qubes-mgmt-salt-vm
 # until we have salt >= 2019.2.2 in dom0
 Requires: python3-pip
 
+# Prevent specific R4.0 overrides to remain
+# during migration process
+Obsoletes: qubes-mgmt-salt-base-overrides <= 4.1.0
```

Comment:
It should be `< 4.1.0` (otherwise not yet existing 4.1.0 would be excluded too, if we ever need to re-introduce it).

Reply:
Sure, I was expecting other first value > 4.1.0